RHOSP-17.1/cinder/powerflex/README.md

```diff
@@ -160,12 +140,12 @@ Install the PowerFlex Storage Data Client (SDC) on all nodes after deploying the
 ### Test the configured Backend
 Finally, create a PowerFlex volume type and test if you can successfully create and attach volumes of that type.
 
-Run the following command to check whether the Cinder service is started. 
+Run the following command to check whether the cinder-volume service is up. 
 ```
 [stack@rhosp-undercloud ~]$ source ~/overcloudrc
 (overcloud) [stack@rhosp-undercloud ~]$ openstack volume service list
```

Comment:
And can we remove this part under "Prepare custom volume mapping for connector configuration"? Since SDC installation is after the overcloud deployment is completed, mentioning it here may cause misunderstanding.
![image](https://github.com/dell/osp-integration/assets/60758925/16673b92-b9f8-413d-bee5-692f0a5078ef)

Reply:
(overcloud) [stack@rhosp-undercloud ~]$ openstack volume create --type powerflex1 --size 8 powerflex_volume1
Can we add volume status verification and expected output at the end? Like:
(overcloud) [heat-admin@elabrhosp85ctl0 ~]$ openstack volume list
+--------------------------------------+-------------------+-----------+------+-------------+
| ID                                   | Name              | Status    | Size | Attached to |
+--------------------------------------+-------------------+-----------+------+-------------+
| 35808e76-c4cd-4ff6-8829-f16c76ebad37 | powerflex_volume1 | available |   8 |             |
+--------------------------------------+-------------------+-----------+------+-------------+
If the PowerFlex backend is ok, the expected output is powerflex_volume1 is available.

Reply:
1. Output added 
2. New section created and SDC install/config moved to this section
3. Output added for checking volume availibility